packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -82,30 +86,39 @@ export const DescriptionList: React.FunctionComponent<DescriptionListProps> = ({
   orientation,
   style,
   ...props
-}: DescriptionListProps) => (
-  <dl
-    className={css(
-      styles.descriptionList,
-      (isHorizontal || isFluid) && styles.modifiers.horizontal,
-      isAutoColumnWidths && styles.modifiers.autoColumnWidths,
-      isAutoFit && styles.modifiers.autoFit,
-      formatBreakpointMods(columnModifier, styles),
-      formatBreakpointMods(orientation, styles),
-      isInlineGrid && styles.modifiers.inlineGrid,
-      isCompact && styles.modifiers.compact,
-      isFluid && styles.modifiers.fluid,
-      isFillColumns && styles.modifiers.fillColumns,
-      className
-    )}
-    style={
-      autoFitMinModifier || style
-        ? { ...(isAutoFit ? setAutoFitMinModifiers(autoFitMinModifier) : {}), ...style }
-        : undefined
+}: DescriptionListProps) => {
+  const listDivRef = React.useRef(null);
+  React.useEffect(() => {
+    if (termWidth && listDivRef) {
```

Comment:
Is the intention to only define the `listDivRef` if `termWidth` is defined?  it looks like the ref is always defined?
If you can only set the width when horizontal can we check for that here?

Reply:
the reference to the list div element will always be defined when the component mounts.  i will add a check for horizontal.